crates/bevy_ui/src/focus.rs

```diff
@@ -44,6 +45,24 @@ pub enum Interaction {
     None,
 }
 
+/// A component storing the position of the mouse relative to the node, (0., 0.) being the bottom-left corner and (1., 1.) being the upper-right
+/// I can be used alongside interaction to get the position of the press.
+#[derive(
+    Component,
+    Deref,
+    DerefMut,
+    Copy,
+    Clone,
+    Default,
+    PartialEq,
+    Debug,
+    Reflect,
+    Serialize,
+    Deserialize,
+)]
+#[reflect(Component, Serialize, Deserialize, PartialEq)]
+pub struct RelativeCursorPosition(Vec2);
```

Comment:
I'm pretty sure this is very useful for more than just slider-widget functionality :)